packages/amazonq/package.json

```diff
@@ -257,25 +257,20 @@
                 }
             ],
             "view/title": [
-                {
-                    "command": "aws.amazonq.startTransformationInHub",
-                    "when": "view == aws.amazonq.transformationHub",
-                    "group": "navigation@1"
-                },
```

Comment:
This `play` was removed a while ago, not sure how it ended up here again.

Reply:
Maybe during merging feature branches it was accidentally added back in.